data/102/1021/en_US.json

```diff
@@ -1,13 +1,13 @@
-{
-	"102": {
-		"1021": {
-			"name": "NOT_NETWORK_ACCOUNT",
-			"message": "Use of this feature requires a linked\nNintendo Network ID.\n\nYou can link an ID from User Settings.",
-			"short_description": "Unknown cause.",
-			"long_description": "Unknown cause.",
-			"short_solution": "Unknown solution.",
-			"long_solution": "Unknown solution.",
-			"support_link": "https://preten.do/102-1021"
-		}
-	}
+{
+	"102": {
+		"1021": {
+			"name": "NOT_NETWORK_ACCOUNT",
+			"message": "Use of this feature requires a linked\nNintendo Network ID.\n\nYou can link an ID from User Settings.",
+			"short_description": "Unknown cause.",
+			"long_description": "Unknown cause.",
+			"short_solution": "Link a Nintendo/Pretendo Network ID to your user.",

```

Comment:
If a short solution is being added, then a long one should also be added. The `short` fields are designed for use by the Discord bot. The `long` fields are used for the website. Only having one of them makes the data unavailable by the other.

You can see a good example of how to format these fields here https://github.com/PretendoNetwork/error-codes/blob/master/data/104/2210/en_US.json

This goes for all cases of this, as there are multiple in this PR